customer/src/test/java/no/unit/nva/customer/get/GetCustomerDoiHandlerTest.java

```diff
@@ -57,7 +57,7 @@ public void beforeEach() {
     void handleRequestReturnsOkWhenARequestWithAnExistingIdentifier() throws IOException, NotFoundException {
 
         var secretDto = existingCustomer.getDoiAgent();
-        var secretDaoArray = "[" + new SecretManagerDoiAgentDao(existingCustomer.getId(), secretDto) + "]";
+        var secretDaoArray = "[" + new SecretManagerDoiAgentDao(secretDto) + "]";
```

Comment:
Since you're doing this a lot, create a small named util that tells us what is going on

Reply:
You can also use ArrayNode to avoid the stringiness :)